spec/01-unit/01-db/11-declarative_lmdb_spec.lua

```diff
@@ -0,0 +1,245 @@
+-- This software is copyright Kong Inc. and its licensors.
+-- Use of the software is subject to the agreement between your organization
+-- and Kong Inc. If there is no such agreement, use is governed by and
+-- subject to the terms of the Kong Master Software License Agreement found
+-- at https://konghq.com/enterprisesoftwarelicense/.
+-- [ END OF LICENSE 0867164ffc95e54f04670b5169c09574bdbd9bba ]
+
+local helpers
+local buffer
+local kong_global
+local conf_loader
+local declarative
+local DB
+
+local kong
+
+
+local ngx_log = ngx.log
+local ngx_debug = ngx.DEBUG
+local lmdb_mlcache
+do
+  local resty_mlcache = require "kong.resty.mlcache"
+  lmdb_mlcache = assert(resty_mlcache.new("lmdb_mlcache", "lmdb_mlcache", {
+    lru_size = 1000,
+    ttl      = 0,
+    neg_ttl  = 0,
+    resurrect_ttl = 30,
+    ipc      = {
+      register_listeners = function(events)
+        ngx_log(ngx_debug, "register lmdb worker events ", tostring(events))
+      end,
+      broadcast = function(channel, data)
+        ngx_log(ngx_debug, "broadcast lmdb worker events ", tostring(channel), tostring(data))
+      end
+    },
+  }))
```

Comment:
always recommend to purge the cache, if might be that someone will leave some dirty data in the same "lmdb_mlcache" mlcache. like:

```
lmdb_mlcache:purge(true)
```

Reply:
Done.